Makefile

```diff
@@ -111,6 +111,7 @@ transfertest: reset
 rewardtest: reset
 	@./scripts/testsend
 	python scripts/reward/testRewards.py
+	make reset
```

Comment:
do we need this becuase testRewards also creates wallet addresses ?

Reply:
> do we need this becuase testRewards also creates wallet addresses ?

Yes. Both create wallet addresses. Without 'reset' the 2nd test somehow doesn't work well on the \keystore\ left by 1st test.

Reply:
Yeah i noticed that as well, if we use the same pub /private key pair to generate two secure walllet accounts they stop working, 
Maybe we should look at it in a different ticket

Reply:
Yep. Can you approve this one and let Mani merge CI first